tests/data/claypit/tests/scenario/DataProviderCest.php

```diff
@@ -47,7 +47,9 @@ public function testDependsWithDataProvider(ScenarioGuy $I, Example $example)
 
        /**
         * @group dataprovider
-        * @depends DataProviderCest:testDependsWithDataProvider
+        * @depends DataProviderCest:testDependsWithDataProvider:1
```

Comment:
You fixed our test, but your change will break tests for everyone else who are using test dependencies with data providers.
As such it can't be merged to 2.3 branch and I am not sure if it could be merged to master.

A better solution would be to make dependency checker ignore iteration suffix (I don't know where it is, it could be in PhpUnit codebase).
Another alternative would be to fix the original issue without making modifications to test signature.

Reply:
Yeah I realised that as soon as I committed it and am looking for the dependency checker at the moment. Any pointers on where to look?

Reply:
https://github.com/Codeception/Codeception/blob/b54eaf4007484f36145c1dc8c64da1874adbc340/src/Codeception/Subscriber/Dependencies.php#L32